docs/pages/setup/guides/joining-nodes-aws-iam.mdx

```diff
@@ -0,0 +1,212 @@
+
+- The `tctl` admin tool version >= (=teleport.version=), which you can download
+  by visiting the
+  [customer portal](https://dashboard.gravitational.com/web/login).
+
+  ```code
+  $ tctl version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+- An AWS EC2 instance to act as a Teleport node, with the Teleport binary
+  installed.
+
+</TabItem>
+<TabItem scope={["cloud"]}
+  label="Cloud">
+
+- A Teleport Cloud account. If you do not have one, visit the
+  [sign up page](https://goteleport.com/signup/) to begin your free trial.
+
+- The Enterprise version of the `tctl` admin tool. To download this, visit
+the [customer portal](https://dashboard.gravitational.com/web/login).
+
+  ```code
+  $ tctl version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+- An AWS EC2 instance to act as a Teleport node, with the Teleport binary
+  installed.
+
+</TabItem>
+</Tabs>
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/4. Set up AWS IAM credentials
+Every node or Proxy using the IAM method to join your Teleport cluster needs AWS
+IAM credentials in order to call the `sts:GetCallerIdentity` API. No specific
+IAM policy or permissions are needed. Any IAM user or role can call this API.
+
+If running your node on an EC2 instance, it is sufficient to attach any IAM
+role to the instance. To attach an IAM role from the EC2 dashboard, select
+`Actions > Security > Modify IAM role`.
+
+You can also provide the IAM credentials to Teleport through a shared
+configuration file or environment variables. For details, see the following guide:
+
+[Specifying Credentials](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials)
+
+## Step 2/4. Create the AWS node joining token
+Under the hood, nodes will prove that they are running in your AWS account by
+sending a pre-signed `sts:GetCallerIdentity` request to the Teleport Auth Server. The
+node's identity must match an allow rule configured in your AWS node joining
+token.
+
+Create the following `token.yaml` with an `allow` rule specifying your AWS
+account and the ARN that your node's identity must match.
+
+```
+# token.yaml
+kind: token
+version: v2
+metadata:
+  # the token name is not a secret because instances must prove that they are
+  # running in your AWS account to use this token
+  name: iam-token
+  # set a long expiry time, the default for tokens is only 30 minutes
+  expires: "3000-01-01T00:00:00Z"
+spec:
+  # use the minimal set of roles required
+  roles: [Node]
+
+  # set the join method allowed for this token
+  join_method: iam
+
+  allow:
+  # specify the AWS account which nodes may join from
+  - aws_account: "111111111111"
+  # multiple allow rules are supported
+  - aws_account: "222222222222"
+  # aws_arn is optional and allows you to restrict the IAM role of joining nodes
+  - aws_account: "333333333333"
+    aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-node-role/i-*"
```

Comment:
just noticed the account number in the arn doesn't match, this is probably more clear (my own mistake from when i originally wrote this)

```suggestion
  - aws_account: "333333333333"
    aws_arn: "arn:aws:sts::333333333333:assumed-role/teleport-node-role/i-*"
```